ui/lib/core/addon/components/form-field.js

```diff
@@ -93,6 +95,11 @@ export default Component.extend({
    */
   valuePath: or('attr.options.fieldValue', 'attr.name'),
 
+  isReadOnly: computed('attr.options.readOnly', 'mode', function() {
+    let readonly = this.attr.options ? this.attr.options.readOnly : false;
```

Comment:
If you want, this could be re-written as `let readonly = this.attr.options?.readOnly || false`

Reply:
Nice!